<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LuaConsole</class>
 <widget class="QMainWindow" name="LuaConsole">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lua Console</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QToolButton" name="btnRunAgain">
      <property name="text">
       <string>Run Again</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>632</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionRun_Script"/>
    <addaction name="actionRunAgain"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionRun_Script">
   <property name="text">
    <string>Run Script...</string>
   </property>
  </action>
  <action name="actionRunAgain">
   <property name="text">
    <string>Run Again</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
